tests/queries_ported/0_stateless/99003_window_function.sql

```diff
@@ -0,0 +1,6 @@
+drop stream if exists v;
```

Comment:
file name `99003...` was existed, you need reassign different number

Reply:
I only find the 99002 used
https://github.com/search?q=repo%3Atimeplus-io%2Fproton+99002&type=code
https://github.com/search?q=repo%3Atimeplus-io%2Fproton+99003&type=code